portfolio/src/main/webapp/commentScript.js

```diff
@@ -0,0 +1,40 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+document.addEventListener('DOMContentLoaded', displayComments(), false);
+
+function displayComments() {
+  fetch('/comment').then(response => response.json()).then(comments => {
+    const commentsContainer = document.getElementById('comments-container');
+    var html = '<table id="commentsTable" width="80%">';
```

Comment:
Instead of doing this string concatenation, if you would like to, you can also explore javascript templating, e.g. handlebars or mustachejs.